app/views/layouts/application.html.erb

```diff
@@ -105,8 +105,30 @@
   </div>
 </div>
 <% unless internal_navigation? %>
+  <script>
+    function toggleFooterVisibility() {
+      const footer = document.querySelector("#footer");
+      if (!footer) {
+        return;
+      }
+      if (window.location.pathname === "/") {
```

Comment:
Our index page isn't just `"/"`, there's a number of other paths for the index (e.g. sorting by top posts for the year is at `"/top/year"`). I think in total there's `['/', '/top/week', '/top/month', '/top/year', '/top/infinity', '/latest']` (someone please chime in if there are others)

Reply:
Applied this suggestion.
Thanks for sharing this list, it looks like it covers all the cases.

Cross-verified from:
https://github.com/forem/forem/blob/84b2f0145af8d6905a480b065f3651cf01877c56/app/assets/javascripts/initializers/initScrolling.js#L222

https://github.com/forem/forem/blob/84b2f0145af8d6905a480b065f3651cf01877c56/app/javascript/packs/homePage.jsx#L13